app/services/record_declarations/retained/mentor.rb

```diff
@@ -5,6 +5,10 @@ module Retained
     class Mentor < ::RecordDeclarations::Base
       include RecordDeclarations::Mentor
       include Retained
+
+      def valid_evidence_types
+        %w[training-event-attended]
```

Comment:
This isn't right. Should be the same as ECT.

Reply:
Why on earth would mentors attend self-study sessions?

Reply:
I will double check with people who ought to know, but from my understanding, the course that mentors and ects are doing is induction into teaching. It makes sense for ECTs to do self-study sessions - after all, there is a lot they have to learn about teaching, from a lot of course materials.

But mentors are experienced teachers. I don't think they get time to do self study on the induction materials, and from the little UR I remember them doing on E&L, I don't think they really want to either. So it seemed weird to me that they would have self study sessions under FIP.

Reply:
And apparently I was wrong, mentors can do self study. The more you know.